.travis.yml

```diff
@@ -13,24 +13,27 @@ cache:
 before_install:
   - if [ ! -z "${GPG_SECRET_KEYS}" ]; then echo ${GPG_SECRET_KEYS} | base64 --decode | ${GPG_EXECUTABLE} --import; fi
   - if [ ! -z "${GPG_OWNERTRUST}" ]; then echo ${GPG_OWNERTRUST} | base64 --decode | ${GPG_EXECUTABLE} --import-ownertrust; fi
+  - mkdir -p ${HOME}/.travis/
+  - wget https://raw.githubusercontent.com/spals/parent-pom/master/.travis/deploy-lib.sh -O ${HOME}/.travis/deploy-lib.sh
+  - wget https://raw.githubusercontent.com/spals/parent-pom/master/.travis/settings.xml -O ${HOME}/.travis/settings.xml
```

Comment:
Instead of copying the `deploy-lib.sh` script and `settings.xml` to each project, we can download them from GitHub.